flutter_local_notifications/lib/src/platform_specifics/android/method_channel_mappers.dart

```diff
@@ -53,6 +53,7 @@ extension AndroidNotificationChannelMapper on AndroidNotificationChannel {
         'ledColorRed': ledColor?.red,
         'ledColorGreen': ledColor?.green,
         'ledColorBlue': ledColor?.blue,
+        'audioAttributesUsage': audioAttributesUsage.value,
```

Comment:
I had to leave a comment to mark the PR where changes are requested. FYI that these additions require updates to the unit tests that validate what gets passed to the platform channel

Reply:
I have created test case for audio attributes

Reply:
Thanks, I pushed a commit directly to avoid back and forth. The commit was done as there was a change to the channel description that wasn't necessary to do